### PR TITLE
Dev sir 1255

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+using System.Text;
+using NBitcoin;
+using NBitcoin.Altcoins;
+
+namespace Swisschain.Sirius.Sdk.Crypto.TransactionSigning.Signers.Helpers
+{
+    public static class BitcoinCashAddressHelper
+    {
+
+        private static readonly string _bc1 = "bc1";
```
Comment (KonstantinRyazantsev) [inline src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs:12]: It doesn't make sense for BCH

File: src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        {
+            try
+            {
+                return BitcoinAddress.Create(address, network);
+            }
+            catch (Exception)
+            {
+                try
+                {
+                    return new BitcoinColoredAddress(address, network).Address;
```
Comment (KonstantinRyazantsev) [inline src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs:23]: We don't need this

File: src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using NBitcoin.Altcoins;
+
+namespace Swisschain.Sirius.Sdk.Crypto.TransactionSigning.Signers.Helpers
+{
+    public static class BitcoinCashAddressHelper
```
Comment (KonstantinRyazantsev) [inline src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs:9]: This duplicates address formatter

Comment (KonstantinRyazantsev) [reply src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs:7]: We do we need this class at all? Why we can't just use the formatter and just pass its result to the BitcoinAddress ctor where it's needed. I would avoid extracting some blockchain-specific types (such as BitcoinAddress) in this library.

Comment (OttoVT) [reply src/Swisschain.Sirius.Sdk.Crypto/TransactionSigning/Signers/Helpers/BitcoinCashAddressHelper.cs]: I made it internal

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**